django/api/services/test/cra_read.py

```diff
@@ -0,0 +1,17 @@
+# Test loading and interpretting a CRA revenue response
+# Assuming you are currently in the directory `django/api/services/test`...
+# To run this test do the following.
+# ```bash
+# cd ..
+# python3 -m test.cra_read
+# ```
+from cra import read
+
+f = open('test/data/EMLI_RESPONSE_FILE_example.txt','r') # Sample file
+data = read(f) # Extract our values
+sin = data[0].get('sin')
+income = data[0].get('income')
+year = data[0].get('year')
+print(f'SIN:    {sin}')
+print(f'Year:   {year}')
+print(f'Income: ${income}')
```

Comment:
Let's create this using a Django TestCase so we can setup a test suite.

Reply:
Not sure what that is but it sounds legit haha. Did you want me to set this up? Or leave it for a separate ticket?

Reply:
It's Django's built in test helper classes: https://docs.djangoproject.com/en/4.0/topics/testing/overview/

with a pit of convention all tests are then picked up an run on `python manage.py test`

Reply:
It's prob better to do that as a part of this ticket since it's testing that functionality.